examples/great3/cgc.yaml

```diff
@@ -16,9 +16,11 @@
 #    and/or other materials provided with the distribution.
 
 # The Great3 control-ground-constant branch configuration file
-# Note: This configuration will not produce images _identical_ to the ones used for Great3, 
+# Note: This configuration will not produce images _identical_ to the ones used for Great3,
 # since the order of random number generation is different.  However, it is intended to
 # produce an _equivalent_ simulation as the one produced for Great3.
+modules:
```

Comment:
I know that this example isn't really part of the tutorial series, but I think it would be nice for some more documentation on the `modules` top-level config type here.  (I think the only other example of this is `examples/des/draw_psf.yaml`.)  Or alternatively, maybe a new tutorial that more simply shows off the `modules` config type (or augment an existing tutorial?).